.overcommit.yml

```diff
@@ -0,0 +1,4 @@
+PreCommit:
```

Comment:
@TwP: I set this as a pre-commit hook in order to tighten the feedback loop. It can certainly be changed to a pre-push hook if you think that would be a better option.